onnxruntime/python/tools/quantization/calibrate.py

```diff
@@ -16,23 +16,18 @@
 
 class CalibrationDataReader(metaclass=abc.ABCMeta):
```

Comment:
The py file seems to bear warning both by flake and pylint.
Maybe we should use flake to align with the pipelines.

Reply:
we are using the https://github.com/google/yapf with pint8 format.